### PR TITLE
Task/access accounting

### PR DESCRIPTION
**Pull request by AlvaroVega**
*Fermín:* Implements issue #350

Feature to print in a file all access log, including:
- StatusCode of request
- UserId
- ServiceId
- ServiceName
- SubServiceId
- SubServiceName
- Action

These logs includes right access (action is OK). But invalid access (wrong token, wrong user, wrong service, and so on) or non allowed actions are not logged.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -56,8 +56,16 @@ config.access = {
     /**
      * Path of the authentication action.
      */
-    path: '/pdp/v3'
-}
+    path: '/pdp/v3',
```
Comment (fgalan) [inline config.js]: Should this PR include documentaiton of the new feature? 

At least the .md explaining the configuration parameters (as config.js has been touched to include new items), but probably in other places explainin the new accounting functionality.

Comment (fgalan) [reply config.js]: In addition, an entry in CNR file should be include

File: README.md [hunk 2]
```diff
@@ -430,14 +430,16 @@ In order to have the proxy running, there are several basic pieces of informatio
     port: 10026
 },
 ```
-* `config.access`: connection information to the selected Access Control PDP API. Includes a `disable` flag, to allow the proxy to work in authentication-only mode. E.g.:
+* `config.access`: connection information to the selected Access Control PDP API. Includes a `disable` flag, to allow the proxy to work in authentication-only mode and a flag to activate accounting access in file. E.g.:
```
Comment (fgalan) [inline README.md]: It is great to include new config items in this section but I think we also need some brief general documentation about the new feature. Maybe a small section (one or two paragraph) explaining how it works and the format used by the access log file.

**Comment by mrutid** [comment]
Determine Origin:

In your request object there is a property called connection, which is a net.Socket object. The net.Socket object has a property remoteAddress, therefore you should be able to get the IP with this 

request.connection.remoteAddress

**Pull request closed without merging**